lib/avail.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'net/http'
+
+module Avail
+  PVTA_BASE_API_URL = 'https://bustracker.pvta.com/InfoPoint/rest'
+
+  # Returns a hash keyed by stop.
+  # Each value is a hash mapping from route direction data to the next time.
+  # The route direction data is route and headsign.
+  def self.next_departures_from(stops, after:)
+    routes = Hash[Route.all.map { |r| [r.avail_id, r] }]
```

Comment:
I'm trying to think if there is a better way to do this.

Reply:
There is!

https://github.com/umts/bojangles/blob/2badc6e83591e7fbb119947ef61d493fa689e7a0/lib/models/departure.rb#L24

Reply:
Well, kinda. That maps the ID rather than the route itself.

Reply:
Cool, that's better than writing `r` so many times.